steem/steem.py

```diff
@@ -111,6 +111,13 @@ class Steem(object):
     # existing connection
     rpc = None
 
+    # This parameter should be dealt with separately to make sure we
```

Comment:
I think its a lot more likely for someone to shoot themselves in the foot because some other part of program changed this without knowledge, than from someone explicitly forgetting to set simulation mode instance when they need it.